modules/graphdb/templates/05_gdb_backup_conf.sh.tpl

```diff
@@ -23,13 +23,19 @@ cat <<EOF >/usr/bin/run_backup.sh
 az login --identity
 
 # Extract GraphDB password from Azure App Configuration
-graphdb_password="\$(az appconfig kv show --endpoint ${app_configuration_endpoint} --auth-mode login --key graphdb-password | jq -r .value | base64 -d)"
+graphdb_password="\$(az appconfig kv show \
+  --endpoint ${app_configuration_endpoint} \
+  --auth-mode login \
+  --key graphdb-password \
+  | jq -r .value \
+  | base64 -d)"
 
-/usr/bin/graphdb_backup admin \$${graphdb_password} ${backup_storage_account_name} ${backup_storage_container_name}
+# Call the backup command, passing $1 and $2 as the storage account and container
+/usr/bin/graphdb_backup admin "\${graphdb_password}" "\$1" "\$2"
```

Comment:
Assign $1 and $2 to variables for clarity. Or read them as arguments.

Reply:
done